lib/manageiq/providers/workflows/engine.rb

```diff
@@ -48,21 +50,18 @@ def self.floe_docker_runner
               "ca_cert"              => "/run/secrets/kubernetes.io/serviceaccount/ca.crt",
               "namespace"            => File.read("/run/secrets/kubernetes.io/serviceaccount/namespace"),
               "task_service_account" => ENV.fetch("AUTOMATION_JOB_SERVICE_ACCOUNT", nil)
-            }
+            }.merge(floe_runner_settings.kubernetes)
 
             Floe::Workflow::Runner::Kubernetes.new(options)
           elsif MiqEnvironment::Command.is_appliance? || MiqEnvironment::Command.supports_command?("podman")
             options = {}
-            if Rails.env.production?
-              options["root"] = Rails.root.join("data/containers/storage").to_s
-            else
-              options["network"] = "host"
-            end
+            options["root"] = Rails.root.join("data/containers/storage").to_s if Rails.env.production?
```

Comment:
the `development` part was so cool that my brain is working overtime to try and do the same thing with production.